NetKAN/Arkas.netkan

```diff
@@ -0,0 +1,25 @@
+{
+	"identifier"	:	"Arkas",
+	"$kref"		:	"#/ckan/kerbalstuff/1158",
+	"spec_version"	:	"v1.4",
+	"license"  	:	"CC-BY-NC-SA",
+	"comment"	:	"PlanetPack",
+	"abstract"	:	"This Mod adds a single planet to the Kerbol system between the orbits of Moho and Eve.  A thick atmosphere on a desert world with 17 unique biomes - Optional Cloud Support",
```

Comment:
`abstract` will be scraped from KerbalStuff and should not be hardcoded. Whenever possible and practical we allow authors to describe their own mods.